vscode/microsoft-kiota/src/extension.ts

```diff
@@ -135,12 +135,14 @@ export async function activate(
         }
     
         let languagesInformation = await getLanguageInformation(context);
+        const pluginName = getPluginName();
         config = await generateSteps(
           {
             clientClassName: openApiTreeProvider.clientClassName,
             clientNamespaceName: openApiTreeProvider.clientNamespaceName,
             language: openApiTreeProvider.language,
             outputPath: openApiTreeProvider.outputPath,
+            pluginName
```

Comment:
Could you confirm with @maisarissi if the request here was to give the sanitized name as an option in the 'quickpick' items as opposed to default to it? Difference between a dropdown option and the prepopulated input option presented here. Not sure makes much difference because the user can still edit the prepopulated option, but just in case this was not the ask.

Reply:
I believe the ask is to pre-polulate the box with the option. The user can always edit it as they wish as outlined in https://github.com/microsoft/kiota/issues/5110

Reply:
@fey101 I confirmed the acceptance criteria before the work started and this fulfills it